tests/python_client/testcases/test_search.py

```diff
@@ -11947,3 +11947,91 @@ def test_hybrid_search_result_order(self):
         res = collection_w.hybrid_search(req_list, WeightedRanker(*weights), 10)[0]
         is_sorted_descend = lambda lst: all(lst[i] >= lst[i+1] for i in range(len(lst)-1))
         assert is_sorted_descend(res[0].distances)
+
+
+class TestSparseSearch(TestcaseBase):
+    """ Add some test cases for the sparse vector """
+
+    @pytest.mark.tags(CaseLabel.L1)
+    @pytest.mark.parametrize("index, index_params", zip(ct.all_index_types[9:11], ct.default_index_params[9:11]))
+    def test_sparse_index_search(self, index, index_params):
+        """
+        target: verify that sparse index for sparse vectors can be searched properly
+        method: create connection, collection, insert and search
+        expected: search successfully
+        """
+        self._connect()
+        c_name = cf.gen_unique_str(prefix)
+        schema = cf.gen_default_sparse_schema(auto_id=False)
+        collection_w, _ = self.collection_wrap.init_collection(c_name, schema=schema)
+        data = cf.gen_default_list_sparse_data()
+        collection_w.insert(data)
+        params = {"index_type": index, "metric_type": "IP", "params": index_params}
+        collection_w.create_index(ct.default_sparse_vec_field_name, params, index_name=index)
+
+        collection_w.load()
+        collection_w.search(data[-1][-1:], ct.default_sparse_vec_field_name,
+                            ct.default_sparse_search_params, default_limit,
+                            check_task=CheckTasks.check_search_results,
+                            check_items={"nq": default_nq,
+                                         "limit": default_limit})
+
+    @pytest.mark.tags(CaseLabel.L2)
+    @pytest.mark.parametrize("index, index_params", zip(ct.all_index_types[9:11], ct.default_index_params[9:11]))
+    @pytest.mark.parametrize("dim", [1, 16, 512, 32768, 524288])
```

Comment:
i think it is enough to cover dim min and max

Reply:
> i think it is enough to cover dim min and max

The maximum vector dimension of sparse vector is uint32max-1, which is too large, so it is not covered. 
uint32max - 1 = 4294967294

Reply:
Changed to max and min dim validation, exception cases smaller than min dim and larger than max dim added later in pr